pandas/io/excel/_base.py

```diff
@@ -279,13 +279,13 @@
 
     .. versionadded:: 1.2.0
 
-dtype_backend : {{"numpy_nullable", "pyarrow"}}, defaults to NumPy backed DataFrames
-    Which dtype_backend to use, e.g. whether a DataFrame should have NumPy
-    arrays, nullable dtypes are used for all dtypes that have a nullable
-    implementation when "numpy_nullable" is set, pyarrow is used for all
-    dtypes if "pyarrow" is set.
+dtype_backend : {{"numpy_nullable", "pyarrow"}}, defaults to ``numpy`` backed ``DataFrames``
```

Comment:
NumPy is the correct capitalization, now?

Reply:
It is, I just wasn't sure if it would be better to go with NumPy or ``numpy``, but like the comment above with DataFrame and pandas, I agree that it would be distracting to make every instance of NumPy a code block.